pkg/metrics/handler.go

```diff
@@ -14,19 +14,25 @@ import (
 type Handler struct {
 	client promv1.API
 
+	resourceQueries map[string]string
```

Comment:
I would prefer that we have a `queries` struct that contains the different types.